src/components/Header/SideNav/index.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+import Flex from 'styled-flex-component'
+import is from 'styled-is'
+
+import CloseButton from './CloseButton'
+import Dropdown from './Dropdown'
+import OuterAnchorItem from './OuterAnchorItem'
+
+const SideNavList = styled.ul``
+
+const SideNavPanel = styled.div.attrs({
+  as: 'nav'
+})`
+  position: fixed;
+  background: ${props => props.theme.colors.blueBg};
+  height: 100vh;
+  width: 100vw;
+  left: 0;
+  top: 0;
+  z-index: ${props => props.theme.zIndexes.header};
+  transform: translateX(100%);
+  transition: transform ${props => props.theme.animations.fast} ease-in-out;
+
+  /* Thinner version' of the navbar */
+  @media screen and (min-width: 600px) and (max-width: 959px) {
+    width: ${remcalc(295)};
+    left: auto;
+    right: 0;
+    z-index: 10;
```

Comment:
why do we need to change the `z-index` here?

Reply:
Oops. I think I only changed it when I was playing around, trying to fix the dropdowns not appearing in front of the page content.

[It used to be the `header` value from the z-index object in our theme.](https://github.com/yldio/yld.io/pull/157/files#diff-3af6432a615e0c766f3ca19b44ffa57fL95) But I don't know why a z-index needed to be applied in the first place.

Reply:
Ah no, [just copy/pasted](https://github.com/yldio/yld.io/pull/157/files/9764ac16c04781d47310dabd99aa86c7c6e70519#diff-3af6432a615e0c766f3ca19b44ffa57fL108). I'll remove it